connectivity/service/src/test/java/org/eclipse/ditto/connectivity/service/messaging/persistence/ConnectionPersistenceActorTest.java

```diff
@@ -1064,9 +1087,13 @@ public void disabledConnectionLogsAreNotEnabledAfterModify() {
 
             // create connection
             underTest.tell(createConnection, getRef());
+            probe.expectMsg(enableConnectionLogs);
             probe.expectMsg(openConnection);
             expectMsg(createConnectionResponse);
 
+            //Close logging which are automatically enabled via create connection
+            underTest.tell(LoggingExpired.of(connectionId, DittoHeaders.empty()), getRef());
```

Comment:
This seems to change the nature of this test. If I understand correctly, this PR should change the behaviour this test expects, so I'd delete this test.

Reply:
With this we want to simulate a connection that has been in use for a longer time and therefore the logging (which is enabled on connection opening) is disabled

Reply:
Then I suggest to rename the test accordingly.

Reply:
I think the added line prepares the actor state (disabled logging) which is required to test if logging is affected by the modify command. No need to rename IMO.